.env.dev

```diff
@@ -1,4 +1,5 @@
 REACT_APP_API_URL=https://dev.api.dfx.swiss/v1
+REACT_APP_DFX_URL=https://dfx.swiss/
```

Comment:
You could add a `.env` file, which will act as a base file. So `REACT_APP_DFX_URL` could be moved there.

Reply:
is it really needed to move it? I don't mind code duplication in an `.env` file

Reply:
No, just a proposition in case you didn't know.

Reply:
didn't know, indeed :D